install-prereqs.sh

```diff
@@ -30,8 +30,8 @@ sh get-docker.sh
 groupadd docker || true
 usermod -aG docker $USER || true
 
-curl https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 -o phantomjs-2.1.1-linux-x86_64.tar.bz2 -L
-tar xvfj phantomjs-2.1.1-linux-x86_64.tar.bz2
+curl https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 -o /phantomjs-2.1.1-linux-x86_64.tar.bz2 -L
```

Comment:
If that's the case please move it into the docker if clause. The read still tell people to run this file as bootstraping SCT

Reply:
Added logic we discussed.